src/axom/sidre/docs/sphinx/sidre_conduit.rst

```diff
@@ -61,9 +60,9 @@ First build top-level groups required by the Blueprint.
    :end-before: _blueprint_restructure_toplevel_end
    :language: C++
 
-Add the node coordinates.  The Groups in ``cds`` will use the Buffers within ``ds``
-as external pointers that have a description: while Sidre and Conduit will use the
-array type and shape information, ``cds`` will not deallocate the memory.
+Add the node coordinates.  The Views in ``tinymesh`` will point to the Buffers within ``nodes``
```

Comment:
Maybe rephrase to something like "The Views under ``tinymesh`` will point to the same Buffers as the Views under ``nodes`` and ``fields`` so that ``tinymesh`` can use the data ...".  The intent being to emphasize that the Buffers are shared, in the same Dataset, but not contained or owned by any Groups.